### PR TITLE
UG/DG fixes

### PR DESCRIPTION
**Pull request by russelltankaimin**
# [UG](https://russelltankaimin.github.io/tp/UserGuide.html)
# [DG](https://russelltankaimin.github.io/tp/DeveloperGuide.html)

### CODE REVIEW

**Review by zichen-3974 — COMMENTED** [review]
image
Is argument multimap meant to have a space here?

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ title: Developer Guide
 ---
 
 ## **Software Overview**
-EduMate is a desktop app designed for NUS students to manage their academic and social lives. It is optimized for use through a Command Line Interface (CLI) while still providing the benefits of a Graphical User Interface (GUI). If you type quickly, eduMate can complete contact and module management tasks faster than traditional GUI apps.
+EduMate is a desktop app designed for NUS students to manage their academic and social lives. It is optimised for use through a Command Line Interface (CLI) while still providing the benefits of a Graphical User Interface (GUI). If you type quickly, EduMate can complete contact and module management tasks faster than traditional GUI apps.
```
Comment (mynameizzhafeez) [inline docs/DeveloperGuide.md:119]:
```suggestion
EduMate is a desktop app designed for NUS students to manage their academic and social lives. It is optimised for use through a Command Line Interface (CLI) while still providing the benefits of a Graphical User Interface (GUI). If the user can type quickly, EduMate can complete contact and module management tasks faster than traditional GUI apps.
```
We might want to remove instances of "you" language in the DG.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1008,9 +1012,9 @@ What you should see:<br>
 
 ##### Extra Explanation:
 
-The command `sort m/d n/a` would mean that `EduMate` would first sort contacts by decreasing number of common modules with you, and tie-break by its lexicographical ordering.
+The command `sort m/d n/a` would mean that EduMate would first sort contacts by decreasing number of common modules with you, and tie-break by its lexicographical ordering.
 
-Hence, Alex Quinn is ranked higher than Amirul as Alex's name is lexicographically smaller than Amirul even though they share the same common modules as you.
+Hence, Alex Quinn is ranked higher than Amirul as Alex's name is lexicographically smaller than Amirul even though they share the same number of common modules as you.
```
Comment (mynameizzhafeez) [inline docs/UserGuide.md:1017]:
```suggestion
Hence, Alex Quinn is ranked higher than Amirul as Alex's name is lexicographically smaller than Amirul even though they share the same number of common modules with you.
```

We are comparing them together, not with the user.

**Review by mynameizzhafeez — APPROVED** [review]
LGTM. Next person take note of changes.

**Pull request merged**